src/ActivityLogServiceProvider.php

```diff
@@ -51,11 +51,12 @@ protected function bootRoutes(): void
             Route::get('/{id}/activities', [
                 'as' => 'activities',
                 'class' => $model,
-                'uses' => ActivityController::class . '@activities',
+                'uses' => ActivityController::class . '@activity',
```

Comment:
why was this changed?

Reply:
I don't recall when/why this was changed, let me and Ambreen know if you encounter any issues with this